test/postgresql.test.js

```diff
@@ -70,7 +89,31 @@ describe('postgresql connector', function() {
   });
 
   it('should run migration', function(done) {
-    db.automigrate('PostWithBoolean', function() {
+    db.automigrate(['PostWithBoolean', 'Expense'], function() {
```

Comment:
`PostWithBoolean` is another model attached to the same datasource. It was already there before for other tests. I only added the `Expense` model.